ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
It looks like in the pom.xml the maven-surefire-plugin sets the system property
```
<systemProperties>
       <property>
                <name>com.arjuna.ats.arjuna.common.propertiesFile</name>
                <value>jbossts-properties.xml</value>
      </property>
 </systemProperties>
```
It would be better to reset to "jbossts-properties.xml" or this is the default value now ?